### ISSUE TITLE
tab order on "take moderation action" screen follows visual flow and shouldn't to be WCAG compliant

### ISSUE THREAD

**Issue by rahaeli** [bluesky-social/ozone#302]
When the "take moderation action" screen is open, the tab order follows the visual flow, as follows:

1. Copy subject to clipboard
2. Switch to profile
3. The drop-down box to switch reports
4. The user PFP
5. The user display name, linking to the repository view
6. The "peek" link
7. Followers link
8. Follows link
9. Posts link
10. Action dropdown
11. Action help button
12. Reason for action textbox
13. Ack-all checkbox
14. Cancel button
15. Submit button
16. Submit-and-next button
17. Switcher for "Moderation history of Account/Entire Account" 
18. Options 
19. Then each individual event and reporter in the moderation history

This is, *technically*, compliant with [WCAG SC 2.4.3](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html), in that visual access, keyboard access, and screenreader access are all the same, but it fails [SC 1.3.2](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence.html) in that the sequence is not meaningful beyond "this is the order things are shown visually on the page": at the very least 1-3 belong after 16 and before 17 to group "information about the report" (as opposed to "information about this account") together. (I also would argue that 4-9 belong after 16, and before the reordered 1-3 + 17-19, but that's a genuine question of workflow and order that does have questions of meaning: if you polled a dozen accessibility experts you'd probably get 22 different answers.)
